services/advertise.js

```diff
@@ -242,21 +246,23 @@ module.exports = (args, cbk) => {
 
               sent.push(paid);
 
+              totalPaid = ceil(Number(sent.reduce((sum, n) => {
```

Comment:
I'd probably put this total paid as a new const that is calculated just before the budget check, removing the let= at the top